book/src/csi-proxy.md

```diff
@@ -0,0 +1,44 @@
+# CSI Proxy
+
+## Status and Releases
+
+**Git Repository:** [https://github.com/kubernetes-csi/csi-proxy](https://github.com/kubernetes-csi/csi-proxy)
+
+**Status:** Beta starting with v0.2.0
+
+Status | Min K8s Version | Max K8s Version
```

Comment:
Similar to the other component pages, can you add a column for where the community binaries are located?

Reply:
added, right now only in staging

Reply:
I will update this after promote to release is available.

Reply:
Let's update the entry here with v1.0

Reply:
updated